lib/portlayer/exec/container.go

```diff
@@ -151,6 +151,20 @@ func (s State) String() string {
 	return ""
 }
 
+// CurrentState returns current state.
+func (c *Container) CurrentState() State {
```

Comment:
Chatted with @hickeng, and the idiom is preferred because of torn write/reads.  Makes sense.  But please do post if there's another reason.